src/components/NFTClaimerOpenseaLink.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+const props = defineProps<{
+  network: string;
+  contractAddress: string;
+  token?: string;
+}>();
+
+const SUPPORTED_NETWORK = ['1', '5'];
+
+function link() {
+  if (!SUPPORTED_NETWORK.includes(props.network)) {
+    return '';
+  }
+
+  return `https://${
```

Comment:
After minting NFT I can't see it on opensea: https://testnets.opensea.io/assets/goerli/0xbb5800439696ddd63fb0689deaec3b5e4a256863/0xa1ae9d1858f53f0f486524afa9adb4f4dcc63e1d080cee6920e6ef25eb49e1ec

Reply:
Fixed